config.yaml.defaults

```diff
@@ -97,6 +97,15 @@ log:
     # log only those that trigger a warning / error
     api_calls: False
 
+    # SQL logging
+    # see e.g. https://docs.sqlalchemy.org/en/20/core/engines.html#configuring-logging
+
+    # if True or "debug", enable SQL echoing.
+    database: False
+
+    # if true or "debug", enable connection pool logging
```

Comment:
`True`? And if `True` is just a synonym for `debug`, I'd remove that latter option. Preferably only one way to spell something! Then I'll merge.

Reply:
That does seem better.